include/stl2/detail/iterator/counted_iterator.hpp

```diff
@@ -296,18 +296,34 @@ STL2_OPEN_NAMESPACE {
 
 	namespace ext {
 		template<Iterator I>
+		requires Copyable<I>
 		constexpr auto uncounted(const I& i)
 		noexcept(std::is_nothrow_copy_constructible<I>::value) {
 			return i;
 		}
 
+		template<Iterator I>
+		requires !Copyable<I>
```

Comment:
This requirement shouldn't be necessary: the above overload implies that this one happens when I doesn't model Copyable.